datarepo-clienttests/src/main/java/utils/KubernetesClientUtils.java

```diff
@@ -152,6 +156,18 @@ public static void buildKubernetesClientObject(ServerSpecification server) throw
     return list.getItems();
   }
 
+  /**
+   * List all the pods in the current namespace
+   *
+   * @return list of Kubernetes pods
+   */
+  public static List<V1Pod> listPodsForNamespace() throws ApiException {
```

Comment:
If we change all the methods to be namespace-aware, then I think this new method and the existing one can combine -- use the namespace property, but still handle the empty string/null case.